### ISSUE TITLE
WARNING couldn't map socket

### ISSUE THREAD

**Issue by Radstake** [homebridge/HAP-NodeJS#847]
Since the upgrade to 1.2.2 it get this error:
WARNING couldn't map socket coming from ::ffff:127.0.0.1:36358 at local address ::ffff:127.0.1.1 to a interface!
It repeating every 30 seconds; seems that it is not coming from a plugin
node version v10.17.0

**Comment by Supereg** [comment]
quick question about your setup. I assume you running it on a macOS while also having the Home App accessing that homebridge instance?

**Comment by Radstake** [comment]
```
Distributor ID: Raspbian
Description:    Raspbian GNU/Linux 9.13 (stretch)
Release:        9.13
Codename:       stretch
```

**Comment by Supereg** [comment]
fixed with https://github.com/homebridge/HAP-NodeJS/commit/52f61fd6b7327f80954a0e98e7eb561bcf822188
will be released with hap-nodejs 0.8.3, which is currently in beta. Will then arrive soon in homebridge.

This warning shouldn't impact you in any way for now.

**Comment by Radstake** [comment]
That is awesome, thanks!
